src/System.Net.Sockets/src/System/Net/Sockets/DisconnectOverlappedAsyncResult.Unix.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Net.Sockets
+{
+    // DisconnectOverlappedAsyncResult - used to take care of storage for async Socket BeginAccept call.
+    internal sealed partial class DisconnectOverlappedAsyncResult : BaseOverlappedAsyncResult
+    {
+        public void CompletionCallback(SocketError errorCode)
+        {
+            base.CompletionCallback(0, errorCode);
+        }
```

Comment:
Alternatively, we could just make the base CompletionCallback method `protected internal` instead of `protected` (or even just `internal` or `public`, since the type itself is already `internal`), and avoid needing to add another method.